tailcfg/tailcfg.go

```diff
@@ -2456,6 +2457,14 @@ type SSHAction struct {
 	// to use remote port forwarding if requested.
 	AllowRemotePortForwarding bool `json:"allowRemotePortForwarding,omitempty"`
 
+	// AllowLocalUnixForwarding, if true, allows accepted connections
+	// to use local unix forwarding if requested.
+	AllowLocalUnixForwarding bool `json:"allowLocalUnixForwarding,omitempty"`
+
+	// AllowRemoteUnixForwarding, if true, allows accepted connections
+	// to use remote unix forwarding if requested.
+	AllowRemoteUnixForwarding bool `json:"allowRemoteUnixForwarding,omitempty"`
+
```

Comment:
Not sure how to actually change these values on the client, could it be part of another project? Currently they seem to default to false.